guides/common/modules/proc_preparing-the-host-group.adoc

```diff
@@ -0,0 +1,22 @@
+[id="preparing-the-host-group_{context}"]
+= Preparing the Host Group
+
+Use this procedure to prepare a host group to convert CentOS 7 machines.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* *>* *Host groups* and click *Create Host Group*.
+. In the *Name* field, enter a name for the host group.
+. In the *Description* field, enter a description of the host group.
+. In the *Lifecycle Environment* field, select `Library`.
+. Optional: In the *Content view* field enter a value for the Content view.
+. For Steps 7 - 11, {Team} does not recommend setting values for this scenario as they might modify the host in ways that are not intended.
```

Comment:
Step 11 is "Switch to the Activation Keys tab", I guess the right number is 10.

Reply:
Fixed.